Sources/Datadog/Utils/SwiftExtensions.swift

```diff
@@ -23,6 +23,10 @@ extension Double {
         }
         return self / divider
     }
+
+    var inverted: Double {
+        return self == 0 ? 0 : 1 / self
+    }
```

Comment:
Although it seems trivial, would be nice to have it tested in [SwiftExtensionsTests.swift](https://github.com/DataDog/dd-sdk-ios/blob/develop/Tests/DatadogTests/Datadog/Utils/SwiftExtensionsTests.swift). Similar for `divideIfNotZero(by:)` - it's missing tests.